src/components/FilterUtils.res

```diff
@@ -1,22 +1,68 @@
-let generateDefaultUrl = dict => {
-  dict
-  ->Js.Dict.entries
-  ->Belt.Array.keepMap(entry => {
-    let (key, val) = entry
-
-    let strValue = RemoteFiltersUtils.getStrFromJson(key, val)
-    if strValue !== "" {
-      Some(`${key}=${strValue}`)
-    } else {
-      None
-    }
+let useFiltersObject = () => {
+  open HyperswitchAtom
+  filtersAtom
+  ->Recoil.useRecoilValueFromAtom
```

Comment:
You can store a different type in Recoil other than string. Can be defined like 

Recoil.recoilAtom<Type> = Recoil.atom(.
  "recoil_name",
 initialValue,
)

Reply:
we want string only